larq_compute_engine/mlir/ir/lce_ops.td

```diff
@@ -35,25 +35,21 @@ TODO
   }];
 
   let arguments = (ins
-    TensorOf<[BF16, F16, F32, F64, I32]>:$input,
-    TensorOf<[BF16, F16, F32, F64, I32]>:$filter,
+    TensorOf<[F32]>:$input,
+    TensorOf<[F32]>:$filter,
     TensorOf<[F32]>:$fused_multiply,
     TensorOf<[F32]>:$fused_add,
 
     I64ArrayAttr:$strides,
-    TF_AnyStrAttrOf<["SAME", "VALID", "EXPLICIT"]>:$padding,
+    TF_AnyStrAttrOf<["SAME", "VALID"]>:$padding,
     DefaultValuedAttr<TF_ConvnetDataFormatAttr, "NHWC">:$data_format,
     DefaultValuedAttr<I64ArrayAttr, "{1, 1, 1, 1}">:$dilations,
-    DefaultValuedAttr<TF_AnyStrAttrOf<["HWIO", "OHWI", "OHWI_PACKED"]>, "HWIO">:$filter_format
+    DefaultValuedAttr<TF_AnyStrAttrOf<["HWIO", "OHWI"]>, "OHWI">:$filter_format
```

Comment:
`OHWI_PACKED` was reserved for the time we start to bitpack the weights during the conversion. so I think we should still keep it.

Reply:
Since it is not yet supported and the tablegen definition doesn't change the forward compatibility of the flatbuffer, I don't see a reason to keep it, or am I missing something?

Reply:
I am starting to work on this next week. having bitpacked weights is necessary for other projects that we will pursue soon. so why should we remove it now and then add again next week?

Reply:
I don't have a strong opinion on this. I added it back in 350d437